doc/source/features/strict-mode.rst

```diff
@@ -0,0 +1,263 @@
+
+* ``MASTER``: The same as ``ENFORCING`` except that the validation of
+  :ref:`explicit table locking <explicit-table-locking>` is not performed.
+  This mode can be used with clusters
+  in which write operations are isolated to a single node.
+
+By default, *enforcing* strict mode is enabled,
+except if the node is acting as a standalone server
+or the node is bootstrapping, then strict mode is disabled.
+
+To select the mode,
+use the :variable:`pxc_strict_mode` variable in the configuration file
+or the ``--pxc-strict-mode`` option during ``mysqld`` startup.
+
+.. note::
+
+   Although it is recommended to start the server with the necessary mode,
+   you can dynamically change it during runtime.
+   For example, to enable permissive strict mode::
+
+      mysql> SET pxc_strict_mode=PERMISSIVE;
+
+Validations
+===========
+
+Strict mode validations are designed to ensure optimal operation
+for common cluster setups that do not require experimental features
+and do not rely on operations not supported by |PXC|.
+If you are planning to disable strict mode,
+you should be aware of the limitations and effects
+that this may have on data integrity.
+
+.. warning:: If an unsupported operation is performed on a node
+   with strict mode disabled (or set to *permissive*),
+   it will not be validated on nodes where it is replicated to,
+   even if the destination node has *enforcing* or *master* strict mode enabled. 
+
+This section describes the purpose and consequences of each validation.
+
+.. contents::
+   :local:
+
+Storage engine
+--------------
+
+|PXC| currently supports replication only for tables
+that use a transactional storage engine (XtraDB or InnoDB).
+When you enable strict mode,
+non-system persistent tables are checked during runtime.
+If a write or alter operation is performed on an unsupported table,
+one of the following happens:
+
+* In case of *enforcing* or *master* mode,
+  an error is thrown and the operation is denied.
+
+* In case of *permissive* mode,
+  a warning is logged and the operation is permitted.
+
+Due to the time that may be required to scan all tables,
+this validation is not performed during startup.
+
+.. note:: Unsupported tables can be converted to use a supported storage engine.
+
+MyISAM replication
+------------------
+
+|PXC| provides experimental support for replication of tables
+that use the MyISAM storage engine.
+Due to the non-transactional nature of MyISAM,
+it is not likely to ever be fully supported.
+
+When you start the server with strict mode enabled,
+it checks the value of the :variable:`wsrep_replicate_myisam` variable.
+By default, this variable is set to ``OFF``.
+However, if MyISAM replication is enabled, one of the following happens:
+
+* In case of *enforcing* or *master* mode,
+  an error is thrown and server startup is halted.
+
+* In case of *permissive* mode,
+  a warning is logged and server startup continues.
+
+.. note:: This validation is not performed during runtime,
```

Comment:
I see you are still using the document to create it which unfortunately has stale information.

"because the :variable:`wsrep_replicate_myisam` variable
 +   cannot be set dynamically." ... is wrong.

From comment section:
DISABLED:
At startup, no check is enforced.
At runtime, setting wsrep_replicate_myisam to ON/OFF is allowed.